PolyEngine/Core/Src/Collections/String.cpp

```diff
@@ -7,6 +7,8 @@ using namespace Poly;
 
 const String String::EMPTY = String();
 
+const String String::WHITESPACES = String(" \t\r\n");
```

Comment:
Good change, but this should static Dynarray<char> since we don't use it as string.  This also shouldnt be in header at all. Just static field in this cpp.